### ISSUE TITLE
import scribefire into scribefire next?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#400]
```
What's the problem?
Unable to import blogs, etc. from Scribefire into Scribefire Next(currently 
v1.8)

I'm not sure if this is what was meant about importing notes, etc. in the 
changes but it definitely doesn't work for me.


What browser are you using?
Chrome and Firefox on Linux


What version of ScribeFire are you running?
Scribefire 3.5.3.3  and Next 1.8

```

-----
Original issue reported on code.google.com by `ehrich.w...@gmail.com` on 25 Jul 2011 at 11:18

**Comment by GoogleCodeExporter** [comment]
```
Did you try a manual import, or are you talking about the auto-import when you 
install ScribeFire Next? If the latter, try version 4 of ScribeFire - it has 
improved importing.
```

-----
Original comment by `cfinke` on 4 Nov 2011 at 2:07
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in Chrome version 4.2
```

-----
Original comment by `scribefi...@gmail.com` on 29 Aug 2013 at 3:15
* Changed state: **Fixed**
